### PR TITLE
Issue1546 formatting for xml and sql

### PR DESCRIPTION
**Pull request by uweschaefer**

### CODE REVIEW

File: factcast-store/src/test/resources/test_schema.sql [hunk 1]
```diff
@@ -1,123 +1,227 @@
...
+CREATE
+  TABLE
+    fact(
+      ser SERIAL PRIMARY KEY,
+      header JSONB NOT NULL,
+      payload JSONB NOT NULL
+    );
+
+# CREATE
+  OR REPLACE FUNCTION notifyFactInsert() RETURNS TRIGGER AS $$ BEGIN PERFORM pg_notify(
```
Comment (BernhardBln) [inline factcast-store/src/test/resources/test_schema.sql:48]: Still wondering what the # sign does here. Comments in SQL would be "--"...

Apply formatter again after fixing this and check what happens...

```suggestion
CREATE OR REPLACE FUNCTION notifyFactInsert() RETURNS TRIGGER AS $$ BEGIN PERFORM pg_notify(
```

File: factcast-store/src/test/resources/test_schema.sql [hunk 2]
```diff
@@ -1,123 +1,227 @@
...
-#
-
-CREATE OR REPLACE FUNCTION notifyFactInsert() RETURNS trigger AS $$
-BEGIN
-  PERFORM pg_notify('fact_insert', json_build_object(
-    'ser', NEW.ser,
-    'header',NEW.header
-  )::text);
-  RETURN NEW;
+# CREATE
```
Comment (BernhardBln) [inline factcast-store/src/test/resources/test_schema.sql:1]:
```suggestion
CREATE
```

File: factcast-store/src/test/resources/test_schema.sql [hunk 3]
```diff
@@ -1,123 +1,227 @@
...
-	last_access 		TIMESTAMP WITH TIME ZONE DEFAULT now() NOT NULL
-);
-
-CREATE TABLE IF NOT EXISTS shedlock(name VARCHAR(64) NOT NULL, lock_until TIMESTAMP NOT NULL,
-    locked_at TIMESTAMP NOT NULL, locked_by VARCHAR(255) NOT NULL, PRIMARY KEY (name));
-
-create table IF NOT EXISTS snapshot_cache (
-	cache_key 		varchar(2048) not null,
+# CREATE
+  CONSTRAINT TRIGGER tr_deferred_fact_insert AFTER INSERT
```
Comment (BernhardBln) [inline factcast-store/src/test/resources/test_schema.sql:64]: Same

```suggestion
CREATE CONSTRAINT TRIGGER tr_deferred_fact_insert AFTER INSERT
```

File: factcast-store/src/test/resources/test_schema.sql [hunk 4]
```diff
@@ -1,123 +1,227 @@
...
+    ON
+    fact DEFERRABLE INITIALLY DEFERRED FOR EACH ROW EXECUTE PROCEDURE notifyFactInsert();
+
+# CREATE
+  SEQUENCE catchup_seq;
```
Comment (BernhardBln) [inline factcast-store/src/test/resources/test_schema.sql:69]:
```suggestion
CREATE SEQUENCE catchup_seq;
```

File: factcast-store/src/test/resources/test_schema.sql [hunk 5]
```diff
@@ -1,123 +1,227 @@
...
+CREATE
+  TABLE
+    catchup(
+      cid BIGINT,
+      ser BIGINT,
+      ts TIMESTAMP
+    );
+
+# CREATE
+  TABLE
```
Comment (BernhardBln) [inline factcast-store/src/test/resources/test_schema.sql:80]:
```suggestion
CREATE TABLE
```

**Review by BernhardBln — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 6]
```diff
@@ -1,637 +1,670 @@
...
+						</dbeaver>
+					</sql>
+					<formats>
+						<format>
+							<excludes>
+								<exclude>**/documentation-docsy/**</exclude>
+							</excludes>
+							<includes>
+								<include>**/*.xml</include>
+								<include>**/*.xsd</include>
```
[the remark this answers is not in the document]
Comment (uweschaefer) [reply pom.xml:441]: if anyone again formats .md files, i'll be on a killing spree... 
they are constantly broken due to 'clever' formatters not understanding forntmatter, mermaid or even simple markdown tables.

[the remark this answers is not in the document]
Comment (uweschaefer) [reply pom.xml:441]: what is there to format regarding property files?

[the remark this answers is not in the document]
Comment (uweschaefer) [reply pom.xml]: It is official: "formatting" property files this way is not a great idea either:

```
diff --git a/factcast-examples/factcast-example-tls-server/src/main/resources/application.properties b/factcast-examples/factcast-example-tls-server/src/main/resources/application.properties
index dcdd955b5..50f0ae150 100644
--- a/factcast-examples/factcast-example-tls-server/src/main/resources/application.properties
+++ b/factcast-examples/factcast-example-tls-server/src/main/resources/application.properties
@@ -1,12 +1,15 @@
+
 spring.datasource.tomcat.max-wait=20000
 spring.datasource.tomcat.remove-abandoned-timeout=360000
 spring.datasource.tomcat.test-on-borrow=true
-# ensure RDS reconnect behavior (issue #53)
+# ensure RDS reconnect
+behavior (issue #53)
 spring.datasource.tomcat.connectionProperties=socketTimeout=20;connectTimeout=10;loginTimeout=10;
 grpc.server.port=9443
 grpc.server.security.enabled=true
 grpc.server.security.certificateChain=localhost.crt
 grpc.server.security.privateKey=localhost.key
 
-# no factcast-accesss.json is provided
+# no factcast-accesss.json
+is provided
 factcast.security.enabled=false
```

[the remark this answers is not in the document]
Comment (BernhardBln) [reply pom.xml]: Hm, while IntelliJ kind of aligns the =, the spotless formatter would mainly remove leading and trailing spaces.

**Pull request merged**